src/Pdf.php

```diff
@@ -299,8 +299,8 @@ protected function processInput($input, $type=null)
     protected function processOptions($options=array())
     {
         foreach ($options as $key=>$val) {
-            // header-/footer-html expect a URL or a file name, so we need to create a tmp file for HTML content
-            if (is_string($val) && preg_match('/^(header|footer)-html$/', $key)) {
+            // header-/footer-html, xsl-style-sheet and user-style-sheet expect an URL or a file name, so we need to create a tmp file for the content
+            if (is_string($val) && preg_match('/^((header|footer)-html|xsl-style-sheet|user-style-sheet)$/', $key) ) {
```

Comment:
@rahxam Looks good, but 2 things:

* Now that the regex here gets a bit lengthy, could you move it to a class constant like the others? E.g. `REGEX_OPTS_TMPFILE`.
* The regex could be a bit shortened to `/^((header|footer)-html|(xsl|user)-style-sheet)$/`, right?